lib/config/executor/container_vm.go

```diff
@@ -81,6 +81,18 @@ type Diagnostics struct {
 	// ExitLogs is a best effort record of the time of process death and the cause for
 	// restartable entities
 	ExitLogs []ExitLog `vic:"0.1" scope:"read-write" key:"exitlogs"`
+
+	// SyslogConfig holds configuration for connecting to a syslog
+	// server
+	SysLogConfig *SysLogConfig `vic:"0.1" scope:"read-only" key:"syslog"`
+}
+
+// SyslogConfig holds the configuration necessary to connect to a syslog server
+type SysLogConfig struct {
```

Comment:
Is it viable to have a different log level for rsyslog than for the local logs? Is it desirable? I ask because we pump out a lot of logging at debug level and I'm not sure people will want to have their syslog servers flooded.

It could be useful if there's a loginsight analytics package for the debug level output. Also of note is the fact that if the remote goes down we will retry dialing for every single log message.